api/manifest.go

```diff
@@ -211,12 +210,12 @@ func getGitHubReleaseAssetForSHA(client gitHubClient, sha string) (fetchedSHA st
 
 // filterManifest filters items in the the given manifest JSON, omitting anything that isn't an
 // item which has a URL beginning with one of the given paths.
-func filterManifest(manifestBytes []byte, paths mapset.Set) (result []byte, err error) {
+func filterManifest(manifestBytes []byte, paths []string) (result []byte, err error) {
```

Comment:
Please make it clear in the description that you also preserved the input order (of paths) in the manifest API, not just the `product` param.

Reply:
I don't think order has any effect (it's just multiple things to filter by; any of which is acceptible).